schemas/yaml/Sense.yml

```
@@ -1,5 +1,5 @@
 $schema: 'http://json-schema.org/draft-07/schema#'
-$id:     'http://schemas.digitallinguistics.io/Sense-1.1.1.json'
+$id:     'http://schemas.digitallinguistics.io/Sense-1.1.2.json'
```

```suggestion
$id:     'http://schemas.digitallinguistics.io/Sense-1.2.0.json'
```

We want this to be a minor version bump rather than a patch.